blockchain/blockchain.go

```diff
@@ -2151,6 +2156,47 @@ func (bc *BlockChain) CloseBlockSubscriptionLoop() {
 	}
 }
 
+// CurrentBlockUpdateLoop updates the current block in the chain for updating read-only node.
+func (bc *BlockChain) CurrentBlockUpdateLoop(pool *TxPool) {
+	refresher := time.NewTicker(1 * time.Second)
+	defer refresher.Stop()
+
+	for {
+		select {
+		case <-refresher.C:
+			// Restore the last known head block
+			head := bc.db.ReadHeadBlockHash()
+			if head == (common.Hash{}) {
+				logger.Warn("Failed to read head block hash")
```

Comment:
Isn't `continue` required in the next line?

Reply:
If head block hash is nil, then it will retrieve hash from backup hash. So, it has to check the further code.